packages/common/src/adapters/user.ts

```diff
@@ -59,6 +59,11 @@ export const userMetadataFromSDK = (
 
     // Required Nullable fields
     bio: user.bio ?? null,
+    twitter_handle: user.twitter_handle ?? null,
```

Comment:
Just confirming we updated the SDK types to include these fields? I think I remember seeing that in the previous PR.

Reply:
yes thanks to your reminder